config/donation.php

```diff
@@ -22,8 +22,11 @@
     | Configure site to use Donation System
     |
     */
-    'is_enabled'   => true,
+    'is_enabled'   => false,
     'monthly_goal' => 100,
+    'yearly_goal'  => 1200,
+    'goal_type'    => 'yearly', // monthly or yearly
```

Comment:
Instead of having both values defined and choosing between them, how about adding a general goal variable that is either used for monthly or yearly goal (defined by `goal_type`)?

Reply:
Great idea